common/build/build-common/api-extractor-report.esm.current.json

```diff
@@ -5,9 +5,7 @@
 	// or alpha.d.ts) rollup file.
 	"mainEntryPointFilePath": "set-mainEntryPointFilePath-directly-in-leaf-config-file",
 	"apiReport": {
-		// NOTE: "alpha" is temporarily not included to reduce noise while there are no uses.
-		//  Once any package with /legacy also uses /alpha, "alpha" should be added in.
-		"reportVariants": ["public", "beta"]
+		"reportVariants": ["public", "beta", "alpha"]
```

Comment:
Looks like we generate API reports for packages even if they don't have a given entry point. That leads to a lot of annoying spam when changing APIs. The removed TODO did say to do it this way though.

Reply:
I think I was suggesting in discussion (weeks after having written that comment) that we just change this for tree as there is just the one case.

Reply:
There are two cases, tree and fluid-framework, but I still think it would be cleaner. I'll make the change.

Reply:
Done.